registry.json

```diff
@@ -1,4 +1,19 @@
 [
+      {
+      "ticker":"NODO",
+      "poolId":"20c00ebe39ed12dbd2f3e84a5b57b85c972d1039cf79c70d893b6227",
+      "social":{
+         "facebook":"",
+         "twitter":"NodoPool",
+         "website":"https://cardanodo.com",
+         "reddit":"Cardanodo",
+         "instagram":"",
+         "youtube":""
+      },
+      "telegram":"cardanodo",
+      "operatorName":"Story [NODO]",
```

Comment:
This needs to be a real name or empty.

Reply:
Hi Leo42, 
I have updated my name. My name is Story, and I have included C for my last name.